src/documentation/0015-node-export-module/index.md

```diff
@@ -37,6 +37,8 @@ const car = {
 
 module.exports = car
 
+//save this file as car.js
```

Comment:
I think we should be much better off making these two different code blocks with the name of the module as a comment at the top